go/enclave/rpc/GetTransaction.go

```diff
@@ -56,7 +56,7 @@ func GetTransactionExecute(builder *CallBuilder[gethcommon.Hash, RpcTransaction]
 	// Unlike in the Geth impl, we hardcode the use of a London signer.
 	// todo (#1553) - once the enclave's genesis.json is set, retrieve the signer type using `types.MakeSigner`
 	signer := types.NewLondonSigner(tx.ChainId())
-	rpcTx := newRPCTransaction(tx, blockHash, blockNumber, index, gethcommon.Big0, signer)
+	rpcTx := newRPCTransaction(tx, blockHash, blockNumber, index, rpc.config.BaseFee, signer)
```

Comment:
Out of curiosity.. do we want to have it configurable? Before it was hardcoded to 0?

Reply:
currently we have a static configured base fee. 
Eventually, it needs to be dynamic. 
The problem here was that this was returning 0